src/components/ProductListCard/ProductListCard.js

```diff
@@ -0,0 +1,152 @@
+import React, { useState, useEffect } from 'react';
+import './ProductListCard.scss';
+import { useNavigate, Link } from 'react-router-dom';
+
+const ProductListCard = props => {
+  const [change, setChange] = useState(true);
+
+  const navigate = useNavigate();
+  const { data } = props;
+  const goToDetail = id => {
+    navigate(`/productdetail/${id}`);
+  };
+
+  const clickLike = (id, isLike) => {
+    console.log(id);
+    console.log(isLike);
+    if (!localStorage.getItem('accessToken')) {
+      return alert('로그인이 필요합니다.');
+    }
+
+    if (props.isLike === false) {
```

Comment:
isLike 변수이름을 보면 이미 isLike변수가 boolean값을 가지고있는걸로 보입니다.
```suggestion
    if (!props.isLike) {
```
not 연산자를 활용해서 위와같이 나타낼 수 있겠습니다